wagtail/documents/views/chooser.py

```diff
@@ -17,7 +17,7 @@
 permission_checker = PermissionPolicyChecker(permission_policy)
 
 
-def get_chooser_context():
+def get_chooser_js_data():
```

Comment:
Could we change ``js`` to ``json`` here?

Reply:
I've made this name to conform with the image's chooser method. So I think it'll be better to change this method for images chooser too. Is it ok?

Reply:
Oh, I see. Yep I think it makes sense to have the same name in both places. Thanks!